src/main/java/com/pubnub/api/models/consumer/history/MessageType.java

```diff
@@ -0,0 +1,24 @@
+package com.pubnub.api.models.consumer.history;
```

Comment:
I'm not sure it belongs to history

Reply:
I propose to add it to com.pubnub.api. We have UserId and SpaceId there.
What do you think?

Reply:
It'll be used across few of the methods, so I guess that's the best option